search_aggs_bucket_geohash_grid_test.go

```diff
@@ -23,6 +23,96 @@ func TestGeoHashGridAggregation(t *testing.T) {
 	}
 }
 
+func TestGeoHashGridAggregation_PrecisionAsString(t *testing.T) {
+	agg := NewGeoHashGridAggregation().Field("location").Precision("2km")
+	src, err := agg.Source()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	data, err := json.Marshal(src)
+	if err != nil {
+		t.Fatalf("Marshalling to JSON failed: %v", err)
+	}
+	got := string(data)
+	expected := `{"geohash_grid":{"field":"location","precision":"2km"}}`
+	if got != expected {
+		t.Errorf("expected\n%s\n,got:\n%s", expected, got)
+	}
+}
+
+func TestGeoHashGridAggregation_PrecisionAsUint(t *testing.T) {
```

Comment:
I think we could leave it as one example with a `string` and another with an `int`.